recipes/pydaily/meta.yaml

```diff
@@ -15,10 +15,16 @@ build:
   noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
 
+# dependencies are being installed in host because of https://github.com/PingjunChen/pydaily/blob/master/setup.py#L5, which is **NOT** recommended practice; see https://github.com/conda-forge/staged-recipes/pull/14476#discussion_r610546800 for details
```

Comment:
```suggestion
# run dependencies are being installed in host because of 
# https://github.com/PingjunChen/pydaily/blob/master/setup.py#L5, 
# which is **NOT** recommended practice
```